nemo/collections/asr/parts/preprocessing/features.py

```diff
@@ -218,22 +218,26 @@ def __init__(
         max_duration=16.7,
         frame_splicing=1,
         exact_pad=False,
-        stft_exact_pad=False,  # TODO: Remove this in 1.1.0
-        stft_conv=False,  # TODO: Remove this in 1.1.0
         pad_value=0,
         mag_power=2.0,
         use_grads=False,
         rng=None,
         nb_augmentation_prob=0.0,
         nb_max_freq=4000,
+        # Deprecated arguments; kept for config compatibility
+        stft_exact_pad=False,  # TODO: Remove this in 1.1.0
+        stft_conv=False,  # TODO: Remove this in 1.1.0
```

Comment:
should we just accept **kwargs?

Reply:
Id like to avoid that. I have some ideas for a good deprecation decorator for generic functions. I'll impl over the next week.